kuma/search/views.py

```diff
@@ -152,7 +152,8 @@ def suggestions(request):
     return HttpResponse(json.dumps(results), content_type=content_type)
 
 
-@cache_page(60 * 60 * 168)  # 1 week.
+@never_cache
+@cache_page(60 * 60 * 168)  # Cache server-side (memcached) for 1 week.
```

Comment:
I think ``@never_cache`` is wrong here, and ``@cache_page`` should be replaced with a standard cache control header. The OpenSearchDescription XML never appears to change.

```
<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
                       xmlns:moz="http:/www.mozilla.org/2006/browser/search/">
<ShortName>MDN Web Docs</ShortName>

<Description>Search MDN Web Docs</Description>
<InputEncoding>UTF-8</InputEncoding>
<Image width="16" height="16" type="image/png">/static/img/favicon32.png</Image>
<Url type="text/html" method="get" template="https://localhost:8000/en-US/search">
  <Param name="q" value="{searchTerms}"/>
  <Param name="w" value="3"/>
  <Param name="qs" value="plugin"/>
</Url>
<Url type="application/opensearchdescription+xml"
     rel="self"
     template="https://localhost:8000/en-US/search/xml"/>
<Url type="application/x-suggestions+json"
     template="https://localhost:8000/en-US/search/suggestions?q={searchTerms}"/>
<moz:SearchForm>https://localhost:8000/en-US/search</moz:SearchForm>
```